packages/app/app/styles/common/defaults.less

```diff
@@ -23,8 +23,26 @@ a:link {
 //TODO make btn a component and move styles there
 .btn {
     border: none;
+    border-radius: 2px;
+    font-size: 14px;
 }
 
 .btn-primary {
     background: linear-gradient(to right,#3697F2,#3570F4);
+    min-width: 120px;
```

Comment:
Does this mess up any of the modal buttons?

Reply:
I checked the modals. We are good. 

![screen shot 2018-02-23 at 1 51 04 pm](https://user-images.githubusercontent.com/2983409/36614515-932babfa-18a2-11e8-9956-e75787956390.png)